### PR TITLE
Update nav to include file

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -386,6 +386,7 @@ include::analytics:partial$nav.adoc[]
  ** xref:install:upgrade-offline.adoc[Performing the Offline Upgrade]
  ** xref:install:upgrade-individual-nodes.adoc[Performing the Single Node Upgrade]
  ** xref:install:upgrade-to-ipv6.adoc[Upgrading IPv6 Clusters]
+ ** xref:install:upgrade-strategy-for-features.adoc[Upgrading to Couchbase Server 6.5]
```
Comment (RichardSmedley) [inline modules/ROOT/nav.adoc:389]: Should this be 
xref:install:upgrade-strategies.adoc[Upgrading to Couchbase Server 6.5]
?

Comment (amarantha-k) [reply modules/ROOT/nav.adoc:389]: no, this is a new file just for features

Comment (RichardSmedley) [reply modules/ROOT/nav.adoc:389]: In that case, the file I named needs its page-alias removed.
Is that in another PR?

Comment (RichardSmedley) [reply modules/ROOT/nav.adoc:389]: ah, it's commented out

**Review by RichardSmedley — APPROVED** [review]
LGTM

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (amarantha-k) [reply modules/ROOT/nav.adoc:389]: confirmed that it's commented already

**Pull request merged**